src/components/v5/shared/SubNavigationItem/SubNavigationItem.tsx

```diff
@@ -25,27 +42,66 @@ const SubNavigationItem: FC<SubNavigationItemProps> = ({
     </>
   );
 
+  const handleSelectElement = () => {
+    onSelectParentFilter?.(option);
+  };
+
+  const isOptionSelected = useMemo(
+    () => options?.some((item) => item.option === option),
```

Comment:
Please, destructure the object

Reply:
it should stay as it is, otherwise the error will appear: 'option' is already declared in the upper scope on line 19 column 3